onnxruntime/core/framework/path_lib.h

```diff
@@ -20,6 +20,17 @@
 #include "core/session/onnxruntime_c_api.h"
 
 using PATH_CHAR_TYPE = ORTCHAR_T;
+using PATH_STRING_TYPE = std::basic_string<ORTCHAR_T>;
+
+inline std::string OrtPathToString(const PATH_CHAR_TYPE* path)
+{
+#ifdef _WIN32
+    std::wstring s{path};
+#else
+    std::string  s{path};
+#endif
+    return std::move(std::string{s.begin(),s.end()});
```

Comment:
>std::string{s.begin(),s.end()}); [](start = 21, length = 32)

You can not put wide characters into std::string. Why do you need wstring?

Reply:
canceled the change.